js/script.js

```diff
@@ -0,0 +1,188 @@
+}
+
+//PROCESS OF GAME
+
+function playerMove(name) {
+    var computerM = computerMove();
+    params.eachRound++;
+    if ((name === 'rock' && computerM === 'paper') ||
+        (name === 'paper' && computerM === 'scissors') ||
+        (name === 'scissors' && computerM === 'rock')) {
+        params.computerScore++;
+        params.score.innerHTML = params.humanScore + ' - ' + params.computerScore;
+        params.progress.push({
+            'round-nb': params.eachRound,
+            'computer-move': computerM,
+            'player-move': name,
+            'result': 'computer won',
+        });
+        params.output.innerHTML = 'COMPUTER WON. You played ' + name + ', computer played ' + computerM;
+        endRound();
+    } else if ((name === 'rock' && computerM === 'scissors') ||
+        (name === 'scissors' && computerM === 'paper') ||
+        (name === 'paper' && computerM === 'rock')) {
+        params.humanScore++;
+        params.score.innerHTML = params.humanScore + ' - ' + params.computerScore;
+        params.progress.push({
+            'round-nb': params.eachRound,
+            'computer-move': computerM,
+            'player-move': name,
+            'result': 'you won',
+        });
+        params.output.innerHTML = 'YOU WON. You played ' + name + ', computer played ' + computerM;
+        endRound();
+    } else {
+        params.output.innerHTML = 'DRAW. You played ' + name + ', computer played ' + computerM;
+        params.progress.push({
+            'round-nb': params.eachRound,
+            'computer-move': computerM,
+            'player-move': name,
+            'result': 'draw',
+        });
+    }
+}
+
+//END ROUND
+
+function endRound() {
+    if (params.humanScore === params.winner || params.computerScore === params.winner) {
+        paramsWin();
+        controllButtons(true);
+        showModal();
+        createModalTable();
+    }
+}
+
+function paramsWin() {
+    if (params.humanScore === params.winner) {
+        params.gameResult.innerHTML = '<span>GAME OVER  - you won</span>' + '<br>' + '<span>Play Again!</span>';
+    } else if (params.computerScore === params.winner) {
+        params.gameResult.innerHTML = '<span>GAME OVER - computer won</span>' + '<br>' + '<span>Play Again!</span>';
+    }
+}
+
+// SHOW MODAL 
+
+function showModal() {
+    var allModals = document.querySelectorAll('.modal');
+    for (var i = 0; i < allModals.length; i++) {
+        allModals[i].classList.remove('show');
+    }
+    document.querySelector('#modal-one').classList.add('show');
+}
+
+//HIDE MODAL 
+
+function hideModal(event) {
+    if (event) {
+        event.preventDefault();
+    }
+    document.querySelector('#modal-one').classList.remove('show');
+}
+
+function CloseModal() {
+    var closeButtons = document.querySelectorAll('.close');
+    for (var i = 0; i < closeButtons.length; i++) {
+        closeButtons[i].addEventListener('click', hideModal);
+    }
+}
+
+CloseModal();
```

Comment:
1) nazwy funkcji piszemy z małej litery, chyba, że to konstruktory, ale tego jeszcze nie przerabialiśmy
2) czemu ta funkcja jest akurat tutaj wywoływana?